src/test/java/com/datalogics/pdf/samples/extraction/TextExtractTest.java

```diff
@@ -87,7 +87,12 @@ public void testDocumentWithNoText() throws Exception {
         assertEquals(Level.INFO, logRecord.getLevel());
 
         // Verify that the output file was not created
-        final Path path = Paths.get(outputUrl.getPath());
+        /*
+         * appropriateOsPath: If the beginning of the string is a slash, then a character, then a colon and another
+         * slash, replace it with the character, the colon, and the slash (leaving the leading slash off).
+         */
+        final String appropriateOsPath = outputUrl.getPath().replaceFirst("^/(.:/)", "$1");
```

Comment:
@datalogics-kam 

I understand your concern but the leading "/" cannot be present before the C:, U:, N:, etc. which happens on Windows when you call the getPath() method alone.

However I did try your suggested alternate method to convert to URL to a URI like we do for our other samples but that does not fix the issue of the leading "/" on Windows.